### DISCUSSION TITLE
is it possible to show popup over an annotation point?

### DISCUSSION THREAD

**Discussion by AhmedAredah** [pelicanmapping/osgearth#2576] [Support Q&A]
I have couple annotation points on the map that I want to show a popup of the map whenever the user hovers over the annotation point. is there a way of doing this? What I am trying to achieve is something like HUD or a tooltip above the annotation point where I display information about the annotation point/label. I would appreciate any help. Thank you.

**Comment by gwaldron** [comment]
Nothing built in to osgEarth. You may be able to create a popup system using ImGui.
